pyxdf/pyxdf.py

```diff
@@ -611,11 +611,12 @@ def _clock_sync(
                 if range_i[0] != range_i[1]:
                     start, stop = range_i[0], range_i[1] + 1
                     e = np.ones((stop - start,))
-                    X = np.column_stack([e, clock_times[start:stop]])
-                    X /= winsor_threshold
-                    y = clock_values[start:stop] / winsor_threshold
+                    X = np.column_stack([e, np.array(clock_times[start:stop]) / winsor_threshold])
```

Comment:
The old code divides e by winsor_threshold, the new code doesn't. Is this intended?

Reply:
Yes, this is intended. It's also why the intercept is multiplied by winsor_threshold after the function returns. I had an embarrassingly hard time trying to get the offset adjustment within robust_fit to work when the ones column was scaled. I settled on leaving it as ones, making the offset adjustment rather straightforward.